x-pack/legacy/plugins/ml/public/data_frame_analytics/pages/analytics_exploration/components/regression_exploration/evaluate_panel.tsx

```diff
@@ -99,14 +112,21 @@ export const EvaluatePanel: FC<Props> = ({ jobId, index, dependentVariable }) =>
 
   return (
     <EuiPanel>
-      <EuiTitle size="xs">
-        <span>
-          {i18n.translate('xpack.ml.dataframe.analytics.regressionExploration.jobIdTitle', {
-            defaultMessage: 'Job ID {jobId}',
-            values: { jobId },
-          })}
-        </span>
-      </EuiTitle>
+      <EuiFlexGroup gutterSize="s">
```

Comment:
Would it be possible to add the job type and status to the Outliers results page too?